### ISSUE TITLE
Claymore ETH dual miner 11.0 SSL devfee

### ISSUE THREAD

**Issue by r-a-n-d-o-m-u-s-e-r-n-a-m-e** [JuicyPasta/Claymore-No-Fee-Proxy#36]
hi,
Claymore ETH dual miner 11.0 trying to mine devfee on SSL protocol in port 5555 so the proxy doesnt work.
Any chance to get it work?
Thank you!

**Comment by drdada** [comment]
Hello, 
Thank you for your feedback. I think Claymore try to protect his future income and improve the cheat detection system. 
With SSL, all the proxies cheats will be unusable.
Do you know if the devfee still uses the same pool as you (-allpools enabled?) ? Do you have a wireshark trace or something ?
I see 2 workarounds:
- Creating a custom SSL cert and implement SSL in the proxy
- Cooking a false root cert while use a known pool name.

**Comment by smitthy09** [comment]
use this one, works without a proxy and removes the fee
https://github.com/smitthy09/claymore-v11.0-devfee-removed

**Comment by plumpi** [comment]
why did you delete smitty09's comment?
I tested his version on two rigs and it seems to work....
running it over the last 12h I saw an increase of about 3%
I am not certain but so far this seems the best way to remove claymore devfee:
https://github.com/smitthy09/claymore-v11.0-devfee-removed

**Comment by trick77** [comment]
> I tested his version on two rigs and it seems to work....

Thou shall not run *.EXEs from random folks from the internet.

**Comment by dukhanov** [comment]
> I tested his version on two rigs and it seems to work....

Said a guy with a github account created 15 minutes ago...

**Comment by drdada** [comment]
Yes, this guy post closed source proxy scam.! Be aware do not download random shit...

**Comment by r-a-n-d-o-m-u-s-e-r-n-a-m-e** [comment]
Hi drdada,
Same pool but another port: I am using ethermine port 4444 and defvee using  ethermine port 5555.
I can install wireshark or any if it helps to you to get the solution.
Thank you!

**Comment by r-a-n-d-o-m-u-s-e-r-n-a-m-e** [comment]
Hi @drdada 
Can you mix your claymore proxy code with https://github.com/mitmproxy/mitmproxy to work with SSL?
Thank you!
